war/pom.xml

```diff
@@ -459,7 +459,8 @@ THE SOFTWARE.
                 <artifactItem>
                   <groupId>org.jenkins-ci.plugins</groupId>
                   <artifactId>jdk-tool</artifactId>
-                  <version>1.0</version>
+                  <!-- TODO https://github.com/jenkinsci/jdk-tool-plugin/pull/49 -->
```

Comment:
Failing tests are fixed with the inclusion of https://github.com/jenkinsci/jdk-tool-plugin/pull/49